jest.config.js

```diff
@@ -33,11 +33,9 @@ module.exports = {
 
   // An array of regexp pattern strings used to skip coverage collection
 
-  // coveragePathIgnorePatterns: [
-
-  //   "\\\\node_modules\\\\"
-
-  // ],
+  coveragePathIgnorePatterns: [
+    'src/core/Table/filters/defaultFilterFunctions.ts',
```

Comment:
I don't want to write tests for each function 😀 So not to reduce our coverage, I am excluding that file.